Makefile

```diff
@@ -1,6 +1,6 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= quay.io/djwhatle/mig-controller:latest
```

Comment:
Using my Quay repo temporarily, which is currently set up to automatically build images automatically from this repo when changes land in any branch (currently we just have 1 branch, master)